qmxgraph/widget.py

```diff
@@ -396,10 +396,15 @@ def _on_drop(self, event):
 
             # Refer to `mime.py` for docs about format
             version = parsed['version']
-            if version == 1:
-                x = event.pos().x()
-                y = event.pos().y()
+            if version not in (1, 2):
```

Comment:
There is the possibility of a cell to be dropped from another application in that case we will will need the version info to properly handle the dropped data.
I feel kind of a ambiguity here.
On one side I see the dropped data to carry some application specific (and required for proper application function) detail.
On another side it could be desired some kind of cross application drag&drop functionality.

I actually have a feeling that drag&drop should be actually handled by the application. Any thoughts about it?